.github/workflows/deploy_api.yml

```diff
@@ -165,12 +119,6 @@ jobs:
       run: |
         rm -rf ./tmp/data/
 
-  finish:
-    runs-on: self-hosted
-    needs:
-      - push-snapshot-to-s3
-
-    steps:
     - name: Maybe trigger covid-projections repo to use new snapshot.
```

Comment:
@mikelehen Here is a good opportunity to improve the naming of this step. I'd like to know what is contingent that causes you to call it "maybe". Could you call it "Propose that covid-projections repo use new snapshot"?

Reply:
How about: 

> Trigger website PR generation for new snapshot if appropriate (master branch, morning build, etc.)